pages/blog/generated-columns-in-crate.html

```diff
@@ -0,0 +1,188 @@
+title: Generated Columns in Crate.IO 0.54.0
+link: https://crate.io/blog/generated-columns-in-crate
+author: Ruslan Kovalov
+description: New Crate.IO release with the Generated Columns feature is available
+created: 2015-12-04
+post_name: generated-columns-in-crate
+status: publish
+post_type: post
+tags: generated columns, generation expression
+category: developernews
+
+Crate are pleased to announce the support of generated columns. The new functionality will be available from the fresh **0.54.0** testing release!
```

Comment:
@seut Is the standard available for anyone to read online? I tried looking, but all I could find was a Wikipedia page that didn't say much and searching the ANSI standards site resulted in no results :(